src/onnx/onnx_parser.cpp

```diff
@@ -149,6 +149,25 @@ instruction_ref onnx_parser::node_info::add_broadcastable_binary_op(const std::s
     return this->add_common_op(op_name, arg0, arg1);
 }
 
+/**
+ * @brief A wrapper for insert_common_args(), which constructs an argument list
+ * and inserts multibroadcast and convert ops to match inputs to a common shape and type
+ * as required.
+ *
+ * Use add_common_op() to match input sizes when inputs may be
+ *  either static or dynamic.
+ *
+ * @param op_name               string; Name of operation (op) to add; valid names are the same as
+ * for make_op()
+ *
+ * @param inputs                vector of instruction_ref.  List of instructions which are the
+ * inputs to the new operator.
+ *
+ * @return instruction_ref      Returns an instruction_ref which is the result of the requested
+ * operation.  This makes the added multibroadcast and convert ops, if any, transparent to the
+ * programmer.
```

Comment:
```suggestion
 * operation.
```
IMO that that line is better placed before docstring for `@param` starts.

Reply:
I re-edited the whole description.